apps/web/src/components/Profile/NftGallery/NoGallery.tsx

```diff
@@ -0,0 +1,83 @@
+import { Button } from '@components/UI/Button';
+import { Trans } from '@lingui/macro';
+import type { Profile } from 'lens';
+import type { FC } from 'react';
+import React, { useState } from 'react';
+import { useAppStore } from 'src/store/app';
+
+import Create from './Create';
+
+type Props = {
+  profile: Profile;
+};
+
+const NoGallery: FC<Props> = ({ profile }) => {
+  const [showCreateModal, setShowCreateModal] = useState(false);
+  const currentProfile = useAppStore((state) => state.currentProfile);
+
+  const isOwner = profile.id === currentProfile?.id;
+
+  return (
+    <div className="py-10 flex space-y-4 flex-col items-center justify-center">
+      <Create showModal={showCreateModal} setShowModal={setShowCreateModal} />
+      <div className="grid grid-cols-3 w-full max-w-sm gap-4">
+        <div className="col-span-2 flex h-[250px] justify-center items-center bg-brand-100 rounded-xl">
+          <img
+            width={100}
+            height={100}
+            className="w-20 h-20"
+            src="https://i.imgur.com/OAauOym.png"
```

Comment:
@bigint we need to upload these images to assets 🙇🏻

Reply:
sure will upload this and share link here

Reply:
Reminder

Reply:
I'll take care of it 🙇🏼